File: SDKs/twilio.com/api/1.34.0/python/src/sdk/models/shared/api_v2010_account_conference.py
from dataclasses import dataclass, field
from typing import Any,Enum,Optional
from dataclasses_json import dataclass_json
from sdk.models import shared


@dataclass_json
@dataclass
class APIV2010AccountConference:
    account_sid: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'account_sid' }})
    api_version: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'api_version' }})
    call_sid_ending_conference: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'call_sid_ending_conference' }})
    date_created: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'date_created' }})
    date_updated: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'date_updated' }})
    friendly_name: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'friendly_name' }})
    reason_conference_ended: Optional[shared.ConferenceEnumReasonConferenceEndedEnum] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'reason_conference_ended' }})
    region: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'region' }})
    sid: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'sid' }})
    status: Optional[shared.ConferenceEnumStatusEnum] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'status' }})
    subresource_uris: Optional[dict[str, Any]] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'subresource_uris' }})
    uri: Optional[str] = field(default=None, metadata={'dataclasses_json': { 'field_name': 'uri' }})
    

File: SDKs/twilio.com/api/1.34.0/go/pkg/models/shared/apiv2010accountconference.go
package shared

type APIV2010AccountConference struct {
	AccountSid              *string                                  `json:"account_sid"`
	APIVersion              *string                                  `json:"api_version"`
	CallSidEndingConference *string                                  `json:"call_sid_ending_conference"`
	DateCreated             *string                                  `json:"date_created"`
	DateUpdated             *string                                  `json:"date_updated"`
	FriendlyName            *string                                  `json:"friendly_name"`
	ReasonConferenceEnded   *ConferenceEnumReasonConferenceEndedEnum `json:"reason_conference_ended"`
	Region                  *string                                  `json:"region"`
	Sid                     *string                                  `json:"sid"`
	Status                  *ConferenceEnumStatusEnum                `json:"status"`
	SubresourceUris         map[string]interface{}                   `json:"subresource_uris"`
	URI                     *string                                  `json:"uri"`
}
